public/data/vikalp-tomer.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Vikalp Tomer",
+    "type": "personal",
+    "bio": "React Developer || Open Source Enthusiast",
+    "avatar": "https://github.com/vikalp-tomer.png",
+    "links": [
+      {
+        "name": "Connect with me on LinkedIn",
+        "url": "https://www.linkedin.com/in/vikalp-tomer/",
+        "icon": "linkedin"
+      },
+      {
+        "name": "Check out my GitHub",
+        "url": "https://github.com/vikalp-tomer",
+        "icon": "github"
+      }
+    ]
+  }
+  
```

Comment:
File is not finished with a blank line, not a blocker, it is marked with red sign here by GitHub - just to make you aware for next time

Reply:
Thanks. Will remember from next time.